### DISCUSSION TITLE
vc-drawings 可以在其他的按钮或者方法调用 开始绘制吗？

### DISCUSSION THREAD

**Discussion by xllyll** [zouyaoji/vue-cesium#523] [General]
vc-drawings 可以在其他的按钮或者方法调用 开始绘制吗？比如我自己的一个button，点击之后就开始画圆

**Comment by zouyaoji** [comment]
https://github.com/zouyaoji/vue-cesium/discussions/516

**Reply by xllyll** [reply]
3Q
